stock_picking_move_package_to_package/models/stock_package_level.py

```diff
@@ -0,0 +1,48 @@
+# Copyright (C) 2023 Cetmix OÜ
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class StockPackageLevel(models.Model):
+    _inherit = "stock.package_level"
+
+    package_dest_id = fields.Many2one(
+        "stock.quant.package",
+        "Destination Package",
+        compute="_compute_package_dest_id",
+        inverse="_inverse_package_dest_id",
+        store=True,
+        readonly=False,
+        check_company=True,
+        domain="[('location_id', 'child_of', parent.location_dest_id), "
+        "'|', "
+        "('company_id', '=', False), "
+        "('company_id', '=', company_id)]",
+    )
+
+    @api.depends("move_line_ids.result_package_id")
+    def _compute_package_dest_id(self):
+        """
+        Compute `package_dest_id` based on `result_package_id` of related move lines.
+        Set `package_dest_id` if all move lines share the same `result_package_id`.
+        """
+        for level in self:
+            result_package_ids = level.move_line_ids.mapped("result_package_id")
```

Comment:
I think there is no need for mapped
```suggestion
            result_package_ids = level.move_line_ids.result_package_id
```